location/detection.go

```diff
@@ -9,10 +9,12 @@ import (
 // Database downloaded from http://dev.maxmind.com/geoip/geoip2/geolite2/
 const Database = "data/GeoLite2-Country.mmdb"
```

Comment:
Code source has nothing common with environment there compiled application is executed

- This config should be configured from CLI argument `--ip-database`
OR
- This config should be loader from `ConfigDirectory + "/GeoLite2-Country.mmdb"`

Reply:
Yes - I am working on that in a separate PR, because this PR is not related to this issue.